paparazzi/src/main/java/app/cash/paparazzi/internal/Renderer.kt

```diff
@@ -79,15 +79,15 @@ internal class Renderer(
       .plusFlag(RenderParamsFlags.FLAG_KEY_DISABLE_BITMAP_CACHING, true)
       .withTheme("AppTheme", true)
 
-    val platformDataRoot = System.getProperty("paparazzi.platform.data.root")
-      ?: throw RuntimeException("Missing system property for 'paparazzi.platform.data.root'")
-    val platformDataDir = File(platformDataRoot, "data")
+    val layoutlibRuntimeRoot = System.getProperty("paparazzi.layoutlib.runtime.root")
+      ?: throw RuntimeException("Missing system property for 'paparazzi.layoutlib.runtime.root'")
+    val buildProp = File(layoutlibRuntimeRoot, "build.prop")
+    val platformDataDir = File(layoutlibRuntimeRoot, "data")
     val fontLocation = File(platformDataDir, "fonts")
-    val nativeLibLocation = File(platformDataDir, getNativeLibDir())
     val icuLocation = File(platformDataDir, "icu" + File.separator + "icudt72l.dat")
     val keyboardLocation = File(platformDataDir, "keyboards" + File.separator + "Generic.kcm")
-    val buildProp = File(environment.platformDir, "build.prop")
```

Comment:
Nice! I missed that the build.prop is shipped inside the runtime jar. With this gone you can completely remove `platformDir` from `Environment` which means that Paparazzi doesn't need to care at all about the local SDK anymore

Reply:
correct, that's the next PR

Reply:
probably also means that the `build-class-next-sdk` test is no longer needed since there isn't any way to point to experimental sdks anymore.